docs/make_work.jl

```diff
@@ -24,11 +25,47 @@ using Documenter, DocumenterCitations
 # Remove the module prefix
 Base.print(io::IO, b::Base.Docs.Binding) = print(io, b.var)
 
+# This is needed to make Documenter.jl follow symlinks, which it is also
+# supposed to be able to do, since there has been a PR about this since 2017:
```

Comment:
I don't understand: yes there is this PR, but it's not merged, so in how is Documenter "supposed to be able to do" this?

Reply:
Well, if even the Julia people say "until 522 is merged". That seemed to me like even they agree with this sentiment.